.github/workflows/job_build_windows.yml

```diff
@@ -33,7 +36,7 @@ jobs:
       CMAKE_GENERATOR: 'Ninja Multi-Config'
       CMAKE_CXX_COMPILER_LAUNCHER: ccache
       CMAKE_C_COMPILER_LAUNCHER: ccache
-      CCACHE_REMOTE_DIR: ""
+      CCACHE_REMOTE_DIR: C:\\mount\\caches\\ccache\\windows2022_x86_64_${{ inputs.build-type }}\\${{ inputs.target-branch }}\\${{ inputs.target-branch != 'master' && '..' || '' }} # Use the same cache for major release branches
```

Comment:
So if I understand correctly, it would resolve into something like this for `releases/2024/4`
* If `inputs.target-branch != master`: `C:\\mount\\caches\\ccache\\windows2022_x86_64_${{ inputs.build-type }}\\release/2024/4\\..}}` -> `C:\\mount\\caches\\ccache\\windows2022_x86_64_${{ inputs.build-type }}\\release/2024}}`?
Did you try it with the release branch?

Reply:
Also I would check if everything resolves correctly with merge queue branch

Reply:
I tried - https://github.com/openvinotoolkit/openvino/pull/26803
It will be resolved to C:\\mount\\caches\\ccache\\windows2022_x86_64_Release/release/2024, and that is expected, I thought we are decided to use the same cache for all minor versions (2024.3, 2024.4, 2024.5).

Reply:
removed that logic, because we enabling the automatic cleanup and no needs to use one cache across all minor releases